pom.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.epam.tc</groupId>
+    <artifactId>alexey_guliev</artifactId>
+    <version>1.0</version>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <checkstyle.version>8.45</checkstyle.version>
+
+        <maven-checkstyle-plugin.version>3.1.2</maven-checkstyle-plugin.version>
+        <suiteFile>src/main/resources/allTests.xml</suiteFile>
+    </properties>
+    <profiles>
+        <profile>
+            <id>defaultAddSubProfile</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <properties>
+            <suiteFile>src/main/resources/addSub.xml</suiteFile>
+            </properties>
+
+        </profile>
+        <profile>
+            <id>multiplyDivProfile</id>
+            <properties>
+                <suiteFile>src/main/resources/multiplyDiv.xml</suiteFile>
+            </properties>
+
+        </profile>
+        <profile>
+            <id>allTestsProfile</id>
+            <properties>
+                <suiteFile>src/main/resources/allTests.xml</suiteFile>
+            </properties>
+
+        </profile>
+    </profiles>
+    <dependencies>
+        <dependency>
+            <groupId>com.epam.tat.module4</groupId>
+            <artifactId>calculator</artifactId>
+            <version>1.0</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/src/main/resources/calculator-1.0.jar</systemPath>
+        </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>6.10</version>
```

Comment:
Why do you use `6.10` version instead of latest?

Reply:
My bad, under https://testng.org/doc/download.html maven section is not up-to-date, and I just copied without thinking
I will replace it with 7.4.0